effects/antidot_effect.py

```diff
@@ -0,0 +1,46 @@
+import message_text_config as msg
+import utils
+from citizens.citizen import Citizen
+from citizens.player import Player
+from game import Game
+
+from effects.effect import Effect
+
+
+class AntidotEffect(Effect):
+    def __init__(self, context: 'Context', name: str,
+                 creator: Citizen) -> None:
+        super().__init__(context, name, creator, -1)
+        self.boost_hp = 1000
```

Comment:
Please, use static values for constants (I've mentioned it in previous comment).
Also, use name convention for constans: "SOME_NAME_CONSTANT"

for example
```

class AntidotEffect(Effect):
    BOOST_HP = 1000
     
    def __init(...)__:
        ...

```

Reply:
Done